script/smart-escrow/SmartEscrowCheck.s.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {Script} from "forge-std/Script.sol";
+import {console} from "forge-std/console.sol";
+
+interface GnosisSafeL2 {
+    function getOwners() external view returns (address[] memory);
+    function getThreshold() external view returns (uint256);
+}
+
+interface SmartEscrow {
+    function BENEFACTOR_OWNER_ROLE() external view returns (bytes32);
+    function BENEFICIARY_OWNER_ROLE() external view returns (bytes32);
+    function DEFAULT_ADMIN_ROLE() external view returns (bytes32);
+    function OP_TOKEN() external view returns (address);
+    function TERMINATOR_ROLE() external view returns (bytes32);
+    function benefactor() external view returns (address);
+    function beneficiary() external view returns (address);
+    function defaultAdmin() external view returns (address);
+    function end() external view returns (uint256);
+    function hasRole(bytes32 role, address account) external view returns (bool);
+    function initialTokens() external view returns (uint256);
+    function owner() external view returns (address);
+    function start() external view returns (uint256);
+    function terminate() external;
+    function vestingEventTokens() external view returns (uint256);
+    function vestingPeriod() external view returns (uint256);
+}
+
+/// @title SmartEscrowCheck
+/// @notice https://optimistic.etherscan.io/address/0x143F5773CFE5613ca94196d557c889134F47CB77
+///         Ensure SmartEscrow is configured correctly
+contract SmartEscrowCheck is Script {
+  SmartEscrow constant SMART_ESCROW = SmartEscrow(0x143F5773CFE5613ca94196d557c889134F47CB77);
+
+  function run() public view {
+    require(block.chainid == 10, "Only OP Mainnet");
+
+    // OP Token is configured correctly
+    require(SMART_ESCROW.OP_TOKEN() == 0x4200000000000000000000000000000000000042);
```

Comment:
Can we add reason strings to all the require statements? If a check fails, it's much easier to determine which one failed when there is a unique revert reason. Even just incrementing numbered strings like `"100"`, `"200"`, etc. is sufficient to find the failing line (similar to the gnosis safe [approach](https://github.com/safe-global/safe-smart-account/blob/12136fd54940892bf43258f86b6e8f9d0eff69ea/contracts/Safe.sol#L313))